test/blackbox-tests/utf8-string-interp.t

```diff
@@ -0,0 +1,22 @@
+
+
+  $ . ./setup.sh
+  $ cat > x.ml <<EOF
+  > let x = {j| Hello, \$()|j}
+  > EOF
+  $ melc -ppx melppx x.ml
+  File "x.ml", line 1, characters 20-23:
+  1 | let x = {j| Hello, $()|j}
+                          ^^^
```

Comment:
Is this on the right location? Seems it's picking `()|` rather than `$()`.

Reply:
Sorry, after reading the title, it's unclear to me if the PR was created to showcase this bad location or to fix it.

Reply:
oh good point, I guess we could move it backwards yet one more char.

The [first commit](https://github.com/melange-re/melange/pull/888/commits/f4f814fa8b90e63c3631fe916fa753ed3c2991cc) shows the wrong location, and the [2nd commit](https://github.com/melange-re/melange/pull/888/commits/3c41146f314f895177b76fc64202a966b7214c7a) fixes it.